docs/code/code-basic-nav.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[code-basic-nav]]
+== Basic navigation
+
+[float]
+==== View file structure and information
+The File tree on the left is the major way for you to navigate through your folder structure.  When you are on a directory, Code also presents a finder-like view on the right. Additionally a file path breadcrumb makes it easy to go back to any parent directory.
+
+[float]
+==== Git History and Blame
+Most recent commits will be displayed on directory view, clicking `View More` or `History` will show full commit history of the current folder or file.
+
+[role="screenshot"]
+image::images/code-history.png[]
+
+Clicking on `Blame` will show the most recent commit per line.
```

Comment:
```suggestion
Clicking *Blame* shows the most recent commit per line.
```

Reply:
Also, add some text to indicate how you got to the screen that the Blame button.